src/backend/executor/spi.c

```diff
@@ -3432,3 +3432,9 @@ SPI_setCurrentInternalTxnMode(bool mode)
 {
 	_SPI_current->internal_xact = mode;
 }
+
+void 
+SPI_get_depth(int *depth)
```

Comment:
why are we getting it as an out argument rather than return value?

Reply:
fixed.